### PR TITLE
Remedy updates

### PR DESCRIPTION
**Pull request by dakre18**
Updated remedy.lic to fix multiple bugs, and add functionality to handle special containers. Added match message for herb containers to workorders.lic and common-crafting.lic, and container settings in base.yaml.

### CODE REVIEW

File: remedy.lic [hunk 1]
```diff
@@ -12,6 +12,7 @@ class Remedy
   include DRCC
   include DRCI
   include DRCA
+  include DRCT
```
Comment (rpherbig) [inline remedy.lic]: ❌ You'll need to add `common-travel` on line 8

**Comment by rpherbig** [comment]
@Sarvatt @jandersson could you review/test this, or ask around in Slack for a volunteer to do so?

File: remedy.lic (hunk 1 above)
Comment (dakre18) [reply remedy.lic]: is there a way to add that while it's in a pull request?

Comment (Sarvatt) [reply remedy.lic]: edit it in your branch aka https://github.com/dakre18/dr-scripts/tree/remedy-updates and it’ll update here

Comment (dakre18) [reply remedy.lic]: Done, let me know if i need to do anything else.

**Pull request merged**